README.md

```diff
@@ -42,3 +42,15 @@ The grand idea would be to get this repo at least to the level of
 [rules_ros](https://github.com/mvukov/rules_ros). In particular:
 
 - It has to be convenient to cross-compile a ROS2 deployment.
+
+## Alternatives
+
+For alternative approaches, see:
+
+<!--
+DNM: Pending merge of https://github.com/RobotLocomotion/drake-ros/pull/194
+-->
+
+- [`RobotLocomotion/drake-ros/bazel_ros2_rules`](https://github.com/RobotLocomotion/drake-ros/tree/0ef58e9b1187388655f315b5c48444ab45dbc478/bazel_ros2_rules/ros2#alternatives), which includes a brief analysis of this
+  and other approaches.
+- [`ApexAI/rules_ros`](https://github.com/ApexAI/rules_ros/)
```

Comment:
How about we keep the list alph-sorted, such that it's more-less clear there is no bias?

Reply:
Done a41b6b7